client/apps/articles_list/routes.coffee

```diff
@@ -15,10 +15,20 @@ Transport = require('lokka-transport-http').Transport
 
   client.query(publishedQuery)
     .then (result) =>
-      res.locals.sd.ARTICLES = result.articles
-      res.locals.sd.CURRENT_CHANNEL = req.user?.get('current_channel')
-      res.render 'index',
-        articles: result.articles || []
-        published: true
-        current_channel: req.user?.get('current_channel')
-    .catch -> next()
\ No newline at end of file
+      if result.articles.length > 0
+        renderArticles res, req, result, true
+      else
+        publishedQuery = query "published: false, channel_id: \"#{channel_id}\""
```

Comment:
It's a little confusing to have `publishedQuery` be a query for unpublished articles -- correct me if i'm wrong but this would automatically display the drafts view instead of published right? what if they click the published tab after the page initially loads?